sphinx-docs/How-to-Build-Agents.md

```diff
@@ -144,6 +144,35 @@ an argument to the agent's spawn command and upon the agent's check in, for this
 agent's profile. The following section explains how lateral movement tracking has been enabled for the default agent,
 Sandcat.  
 
+## Compiling Sandcat Agent
```

Comment:
Can this heading be improved? Maybe `Manually Compiling Sandcat` or `Compiling Preconfigured Sandcat Agent` or something else more descriptive?

Reply:
Fixed!